### PR TITLE
Fix image for OpenShift

### PR DESCRIPTION
**Pull request by fpob**

### CODE REVIEW

**Review by thaorell — APPROVED** [review]

**Review by daniellutz — APPROVED** [review]

**Review by brhatiga — DISMISSED** [review]

File: containers/Dockerfile [hunk 1]
```diff
@@ -4,13 +4,17 @@ WORKDIR /opt/app-root/src
...
 
 # https://catalog.redhat.com/software/containers/ubi8/nginx-118/5f521a6f9dd2d5ca7158e5dc
 FROM registry.access.redhat.com/ubi8/nginx-118
 COPY --from=build /opt/app-root/src/dist .
 COPY ./containers/nginx.conf /opt/app-root/etc/nginx.default.d/nginx.conf
 COPY ./containers/nginx-entrypoint /opt/app-root/entrypoint
-RUN /opt/app-root/entrypoint
+
+# https://docs.openshift.com/container-platform/4.11/openshift_images/create-images.html#images-create-guide-openshift_create-images
+RUN touch /opt/app-root/src/index.html \
```
Comment (thaorell) [inline containers/Dockerfile:15]: I get `#16 0.390 chgrp: changing group of '/opt/app-root/src/index.html': Operation not permitted`

**Review by thaorell — CHANGES_REQUESTED** [review]

File: containers/Dockerfile (hunk 1 above)
Comment (fpob) [reply containers/Dockerfile:15]: I don't see such error in GitHub pipeline or when I try docker and podman locally. What do you use to build images?

Comment (thaorell) [reply containers/Dockerfile:15]: I'm using Docker on Mac OS X

Comment (fpob) [reply containers/Dockerfile:15]: Probably an issue in MacOS. It is well known that Docker does not work well on non-Linux systems.

Comment (thaorell) [reply containers/Dockerfile:15]: @fpob the github pipeline doesn't use docker build though. I'd like an alternative that works on my computer too if possible

Comment (thaorell) [inline containers/Dockerfile:15]: try 
```
USER root
RUN touch /opt/app-root/src/index.html \
    && chgrp 0 /opt/app-root/src/index.html \
    && chmod g=u /opt/app-root/src/index.html
```
I tested this locally on Mac OS X and on stage

**Review by thaorell — CHANGES_REQUESTED** [review]

**Review by thaorell — APPROVED** [review]

**Pull request merged**